assets/src/eejs/index.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Exported to the `eejs` global.
+ */
+
+/**
+ * This will hold arbitrary data assigned by the Assets Registry.
+ * @type {{}}
+ */
+export let data = eejsdata.data || {};
```

Comment:
`eejsdata` is provided by `EventEspresso\core\services\assets\Registry` as a part of the data api.  Thus on page load, `eejs.data` will automatically be populated by whatever data was registered.   Things are done this way because `eejs.data` itself is protected and cannot be manipulated outside of the global (which I _think_ is probably a good thing?)

Reply:
Two things:

- all bundles are scoped to the bundle (unless exposed on a global).  This not only is scoped to a bundle but it is also declared with `let` so the scope will be limited to the object its attached to.

Reply:
Just making note that Brent and I talked this out in slack and we're leaving as is.